Makefile

```diff
@@ -247,31 +257,30 @@ ci-check-working-copy: generate
 .PHONY: ci-wasm
 ci-wasm: wasm-test
 
-.PHONY: build-docker
-build-docker: ensure-release-dir
-	$(GO) build $(GO_TAGS) -o $(RELEASE_DIR)/opa_docker_$(GOARCH) -ldflags $(LDFLAGS)
-
 .PHONY: build-linux
 build-linux: ensure-release-dir
-	@$(MAKE) build GOOS=linux CGO_ENABLED=0 WASM_ENABLED=0
+	@$(MAKE) build GOOS=linux
+	chmod +x opa_linux_$(GOARCH)
 	mv opa_linux_$(GOARCH) $(RELEASE_DIR)/
 
 .PHONY: build-darwin
 build-darwin: ensure-release-dir
-	@$(MAKE) build GOOS=darwin CGO_ENABLED=0 WASM_ENABLED=0
+	@$(MAKE) build GOOS=darwin
+	chmod +x opa_darwin_$(GOARCH)
 	mv opa_darwin_$(GOARCH) $(RELEASE_DIR)/
 
 .PHONY: build-windows
 build-windows: ensure-release-dir
-	@$(MAKE) build GOOS=windows CGO_ENABLED=0 WASM_ENABLED=0
+	build/ensure-windows-toolchain.sh
```

Comment:
Just thinking out loud...is the intention here to relegate `build-linux`, `build-darwin`, and `build-windows` to CI usage? If so, I wonder if it would make sense to just make copies of these targets (name them `ci-build-linux`, etc.) and then echo a message in the old targets telling users to update and use `ci-build-darwin`, etc. if they actually want to cross-compile. For non-cross-compile use cases, users can just run `make build` (so there's no reason to be running `make build-linux`, etc.) Trying to avoid breaking things. Basically just print loud messages that tell people to use other targets and that these targets could go away in the future.